openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2236,6 +2215,118 @@ The following is a non-normative example of a Credential Request with Credential
 
 The value of the `credential` claim in the Credential Response MUST be a string that is an SD-JWT VC. Credentials of this format are already suitable for transfer and, therefore, they need not and MUST NOT be re-encoded.
 
+# Claims Description {#claims-description}
+
+A claims description object is an object used to describe how a certain claim in
+the Credential should be displayed to the End-User. It is used in the `claims`
+or `credentialSubject` parameters in the Credential Issuer metadata defined in
+(#format-profiles).
+
+For a claims description object, the following keys defined by this
+specification MAY be used to describe the claim or claims (other keys MAY also
+be used):
+
+  * `path`: REQUIRED. Array. Claim path query as defined in (#claims-path-query)
+    to identify the claim(s) in the Credential.
+  * `mandatory`: OPTIONAL. Boolean which, when set to `true`, indicates that the
+    Credential Issuer will always include this claim in the issued Credential.
+    If set to `false`, the claim is not included in the issued Credential if the
+    wallet did not request the inclusion of the claim, and/or if the Credential
+    Issuer chose to not include the claim. If the `mandatory` parameter is
+    omitted, the default value is `false`.
+  * `value_type`: OPTIONAL. String value determining the type of value of the
```

Comment:
I feel that `value_type` needs more description. 
Is the default `string`?
The following value_types that are needed by our wallets are
1. iso8601_date
2. iso8601_time
3. iso8601_datetime
4. iso8601_duration
5. uri
6. image/* where the credential has a generic image type like the HTTP accepts header

Reply:
Please file this request for finer-grained value types as a separate issue.

Reply:
#425